app/src/main/java/com/imdb_compose/ui/PersonDetailsPage.kt

```diff
@@ -0,0 +1,176 @@
+package com.imdb_compose.ui
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Add
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonColors
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewModelScope
+import androidx.navigation.NavController
+import coil.compose.AsyncImage
+import com.imdb_compose.BottomBar
+import com.imdb_compose.domain.Retrofit
+import com.imdb_compose.TopBarWithBackBtn
+import com.imdb_compose.gray500
+import com.imdb_compose.isLoading
+import com.imdb_compose.ripeMango
+import com.imdb_compose.shadow
+import kotlinx.coroutines.launch
+
+@SuppressLint("CoroutineCreationDuringComposition", "StateFlowValueCalledInComposition")
```

Comment:
You should avoid suppressing lint warnings. They are usually a signal you're doing something wrong. If you are going to suppress them, you should provide a reason next to each rule as to why this warning isn't helpful